task/buildah/0.2/buildah.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   labels:
-    app.kubernetes.io/version: "0.2.1"
+    app.kubernetes.io/version: 0.2.1
```

Comment:
Do we want to be consistent with which is first, labels or annotations?

Reply:
:shrug:

Reply:
Either way it's out of scope in this PR.